src/components/header/settingsButton/index.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import {useNavigation} from "@react-navigation/native";
 import {Routes} from "../../../constants";
-import Button, {ButtonType} from "../../Button";
+import {Button} from "../../";
```

Comment:
This looks not obviously....but left it for lection about absolute path ;)

Reply:
Refactored to use absolute path with aliases